src/core/utils/array.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Same as the classic array.flat() method, which I cannot use
+ * because that does not pass some CI pipelines. I'm guessing
+ * there is somehow differently configured transpilation.
+ * Didn't have time to get into the details.
+ */
+export const arrayFlat = (arrayOfArrays) => {
+  return arrayOfArrays.reduce((acc, item) => {
+    return acc.concat(item)
+  }, [])
+}
```

Comment:
instead of inlining this utility, please use https://npmjs.com/array.prototype.flat.

Transpilation only changes syntax; array methods are API.